src/shop-redux-store.js

```diff
@@ -0,0 +1,19 @@
+import createStore from '../node_modules/@0xcda7a/redux-es6/es/createStore.js';
+import applyMiddleware from '../node_modules/@0xcda7a/redux-es6/es/applyMiddleware.js';
+import origCompose from '../node_modules/@0xcda7a/redux-es6/es/compose.js';
+import thunk from '../node_modules/redux-thunk/es/index.js';
+
+const compose = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || origCompose;
+
+const reducers = {};
+
+export function installReducers(r) {
+  Object.assign(reducers, r);
+}
+
+export const store = createStore(
+  (state, action) => {
+    const r = reducers[action.type];
```

Comment:
Associating a reducer directly with the action is likely not a good pattern to codify like this in general. (See criticism of Ducks pattern.)

Reply:
This is not exactly a reducer since it only ever processes one action type. Probably should change this or not call this a reducer.